app/matplotlib.ipynb

```diff
@@ -85,24 +110,44 @@
    }
   },
   {
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-10T09:28:34.563439Z",
+     "start_time": "2024-11-10T09:28:34.468603Z"
+    }
+   },
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "# write your code here\n",
+    "plt.scatter(x, y ,c=colors, s=sizes)\n",
+    "\n",
+    "plt.xlabel(\"random X\")\n",
+    "plt.ylabel(\"random Y\")\n",
+    "plt.title(\"random scatter plot\")\n",
+    "\n",
+    "plt.colorbar()\n",
```

Comment:
The colorbar is added, but it lacks a label. Consider using `plt.colorbar(label='Color Intensity')` to provide context for the color mapping.